### ISSUE TITLE
Accessibility importer missing areas in some cases

### ISSUE THREAD

**Issue by Rikuoja** [GispoCoding/geoviz-notebooks#47]
**Describe the bug**
The accessibility importer does not always import data for the whole bounding box. Some areas within the bounding box may be left out. This must be a feature of the https://github.com/gboeing/osmnx library .

Example from the Stockholm region:
<img width="674" alt="image" src="https://user-images.githubusercontent.com/9367712/149761275-3181187d-43c0-4c83-9c38-1d1587a5828a.png">

**Comment by Rikuoja** [comment]
The graph contains no nodes in areas served by a single bridge: 
<img width="598" alt="image" src="https://user-images.githubusercontent.com/9367712/149762619-3d078d15-a523-4309-ab4e-6d1f34bf6931.png">

**Comment by Rikuoja** [comment]
Also an area served by three paths, and islands not connected to the graph, are discarded: 
<img width="517" alt="image" src="https://user-images.githubusercontent.com/9367712/149763056-6abb0b44-4a61-4154-a2e9-e16cbf3572bd.png">
